### DISCUSSION TITLE
Change adblock url

### DISCUSSION THREAD

**Discussion by caywin25** [qutebrowser/qutebrowser#6297] [Q&A] [answered]
Hello, 

Thank you for your work.
Is it possible to change adblock URL for specify a personnal url ?

Thank you very much

**Comment by Jan-PieterBaert** [comment, marked as answer]
Using `:set content.blocking.adblock.lists` or `:set content.blocking.hosts.lists` depending on which adblocking you mean.

**Reply by caywin25** [reply]
Perfect :)

Thank you
